modules/migration-installing-legacy-operator.adoc

```diff
@@ -48,15 +48,15 @@ ifdef::installing-3-4,installing-restricted-3-4[]
 [source,terminal,subs="attributes+"]
 ----
 $ sudo podman cp $(sudo podman create \
-  registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:v1.5.1):/operator-3.7.yml ./
+  registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:v1.5.3):/operator-3.7.yml ./
```

Comment:
@apinnick we can remove `registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:v1.5.3):/operator-3.7.yml ./` , as MTC doesn't officially support 3.7.

Reply:
Good point. I'll make sure it's not in the compatibility table, too.